jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BottomNavigationTests.java

```diff
@@ -27,13 +27,13 @@ public void beforeTest() {
     @Test
     public void baseFunctionalTest() {
         bottomNavigationColor.is().displayed();
-        bottomNavigationColor.is().buttonColor(1, Colors.BLACK_TRANSPARENT_06.value());
-        bottomNavigationColor.select(1);
-        bottomNavigationColor.is().buttonColor(1, Colors.BLUE_DARKEN_2.value());
-        bottomNavigationColor.select(2);
-        bottomNavigationColor.is().buttonColor(2, Colors.BLUE_DARKEN_2.value());
-        bottomNavigationColor.select(3);
-        bottomNavigationColor.is().buttonColor(3, Colors.BLUE_DARKEN_2.value());
+        bottomNavigationColor.is().buttonColor("Recents", Colors.BLACK_TRANSPARENT_06.value());
```

Comment:
you can specify collection of test data and iterate through it to get short and clear test which can simply supported in the future.

Reply:
However colors are changing so we are talking just about text values in this case